administrator-guides/permissions/README.md

```diff
@@ -1,88 +1,113 @@
 # Permission
 
-Permission changes are immediate and transparent.  Users do not need to logout, login or refresh to change permissions.
+Rocket.Chat has a vast and customizable permissions system. Most of the actions a user can make requires a certain permission so you can customize what users can do and see.
 
-## Roles Available
+You can find the permission grid under Administration -> Permissions.
 
-- admin
-- moderator
-- owner
-- user
-- bot
-- guest
-- livechat-agent
-- livechat-manager
-- livechat-guest
+Permission changes are immediate and transparent. Users do not need to logout, login or refresh to change permissions.
+
+## Roles
+
+A `Role` is a set of permissions that can be given to users. Rocket.Chat already comes with a set of default roles that can be edited. You can also create new roles for users.
+
+When someone is registered in a Rocket.Chat server, it will normally receive the `user`. If the user is a bot or a guest from a livechat channel, it can receive different roles. Also the first user of a server always receives the role of `admin`.
+
+Roles can have different scopes. Currently we have two scopes `Global` and `Rooms`. The permissions of the `Global` roles are valid for the whole server, while the `Rooms` roles only applies to individual channels, where a user have to be elected to that role in that channel.
```

Comment:
`Global` scope 
`Rooms` scope

Since if i'm not mistaken a role can actually have global and room specific in same role